main.go

```diff
@@ -174,3 +174,38 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func GetCacheNamespaces() map[string]cache.Config {
+
+	// Obtain in-cluster config
+	cfg, err := rest.InClusterConfig()
+	if err != nil {
+		setupLog.Error(err, "error getting in-cluster config")
+		os.Exit(1)
+	}
+
+	// Create the client
+	cli, err := client.New(cfg, client.Options{Scheme: scheme})
+	if err != nil {
+		setupLog.Error(err, "error creating client")
+		os.Exit(1)
+	}
+
+	configmap, err := controllers.GetOdfConfigMap(context.Background(), cli, setupLog)
+	if err != nil {
+		setupLog.Error(err, "error getting configmap")
+		os.Exit(1)
+	}
+
+	namespaces := controllers.GetTargetNamespaces(setupLog, configmap)
+
+	defaultNamespaces := map[string]cache.Config{
+		"openshift-storage-extended": {},
+	}
+
+	for _, namespace := range namespaces {
+		defaultNamespaces[namespace] = cache.Config{}
+	}
```

Comment:
I think you are missing the operator's namespace

Reply:
No. It is part of it.